Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerFrequentConfigurationIssues.ps1

```diff
@@ -349,6 +349,26 @@ function Invoke-AnalyzerFrequentConfigurationIssues {
         Add-AnalyzedResultInformation @params
     }
 
+    if ($null -ne $exchangeInformation.TransportService) {
+        if ($exchangeInformation.TransportService.MaxPerDomainOutboundConnections -lt 40) {
+            $params = $baseParams + @{
+                Name             = "MaxPerDomainOutboundConnections"
+                Details          = "Value set to $($exchangeInformation.TransportService.MaxPerDomainOutboundConnections), which is less than the recommended value of 40. `r`n`t`tMore details: https://aka.ms/TransportRetryConfig"
```

Comment:
aka.ms links should have `HC-` to start off with, so we know they are coming from. Was this link created for Health Checker?

Reply:
No, but is the standard flow to create these HC links? It appears they all point to health checker doc (referencing random ones from the script). Do I just create a new readme page for this check and use the HC link instead?

Reply:
Not all of them go to the docs page, but most of them do. You can decide if you want to create a page for this or just point it to the public doc.

Reply:
Created the read me page but need the aka.ms link published. Do you want to create that?

Reply:
https://aka.ms/HC-TransportRetryConfigCheck

Reply:
you are also going to need to add the file path to the yml file. 

Run the `docs.ps` in the `.build` folder, it will show you.